recipe/meta.yaml

```diff
@@ -26,22 +26,18 @@ requirements:
     - setuptools
     - setuptools_scm
     - cython
-    - numpy 1.10.*   # [not (win and py>=36)]
-    - numpy 1.12.*   # [win and py>=36]
+    - numpy 1.14.*
     - six
-    - arrow-cpp 0.10.0
-    - parquet-cpp 1.5.0.pre
+    - arrow-cpp {{ version }}
 
   run:
     - python
     - setuptools
-    - {{ pin_compatible('numpy', lower_bound='1.10') }}   # [not (win and py>=36)]
-    - {{ pin_compatible('numpy', lower_bound='1.12') }}   # [win and py>=36]
+    - {{ pin_compatible('numpy', lower_bound='1.14') }}
     - pandas
     - six
-    - arrow-cpp 0.10.0
-    - parquet-cpp 1.5.0.pre
     - futures  # [py27]
+    - arrow-cpp {{ version }}
```

Comment:
Do we need to pin the parquet-cpp metapackage? I will merge this as is, but if we need that (to make sure the right binaries get loaded) feel free to open another PR